### PR TITLE
[NCL-2043] added commit hash to the information endpoint

### PR DESCRIPTION
**Pull request by janinko**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -64,6 +64,7 @@
         <version.bc.rest>0.3</version.bc.rest>
         <version.aether>0.9.0.M2</version.aether>
         <version.galley>0.11.3</version.galley>
+	<commit.id.abbrev></commit.id.abbrev>
```
Comment (jomrazek) [inline pom.xml]: formatting

**Comment by janinko** [comment]
I would delete the ``common/src/main/java/org/jboss/da/common/Constants.java`` file as it is generated from template. When you delete it add this file to the .gitignore.

**Comment by janinko** [comment]
Also please reforamt the .xml files. We use 4 spaces for a tab.

File: pom.xml (hunk 1 above)
Comment (janinko) [inline pom.xml]: Is the empty property necessary?

File: common/pom.xml [hunk 2]
```diff
@@ -45,8 +45,34 @@
     </dependencies>
 
     <build>
-        <plugins>
+        <plugins>
```
Comment (janinko) [inline common/pom.xml]: Spaces at the end of the line.

File: pom.xml [hunk 3]
```diff
@@ -64,6 +64,7 @@
         <version.bc.rest>0.3</version.bc.rest>
         <version.aether>0.9.0.M2</version.aether>
         <version.galley>0.11.3</version.galley>
+        <commit.id.abbrev></commit.id.abbrev>
```
Comment (janinko) [inline pom.xml]: Is the empty property necessary?

File: pom.xml [hunk 4]
```diff
@@ -338,9 +339,18 @@
...
+
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>true</filtering>
+                <includes>
+                    <include>**/*.properties</include>
+                </includes>
+            </resource>
+        </resources>
```
Comment (janinko) [inline pom.xml]: Is this necessary?

**Comment by janinko** [comment]
:+1:

**Comment by janinko** [comment]
You need to update test ``RestApiIndexTest.testIndexHtml()``

**Pull request merged**